client/src/components/NodesEntriesSelector/NodesEntriesSelector.scss

```diff
@@ -0,0 +1,11 @@
+.nodes-entries {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+
+  &__button-container {
+    display: flex;
+    justify-content: center;
```

Comment:
Do we need separate classes for application-entries and node-entries? Perhaps we can add a footer which can be used as a button container in both places?

Reply:
Although it's not a big deal